### DISCUSSION TITLE
How to install ?

### DISCUSSION THREAD

**Discussion by ghost** [laksjdjf/IPAdapter-ComfyUI#33] [Q&A] [answered]
Hello !

I don't understand how to install, others nodes have a .zip in release section on github.

If I download files one by one and put them in a folder inside ComfyUI_windows_portable\ComfyUI\custom_nodes all my nodes becomes red.

**Comment by laksjdjf** [comment, marked as answer]
it can be obtained at

![a](https://github.com/laksjdjf/IPAdapter-ComfyUI/assets/22386664/e9a7642c-7552-4144-8622-dfe2ad7c17d3)

**Comment by ghost** [comment]
Thanks a lot ! It works !
